Assignment - Browsers - Week 1 <Mohammed Othman>

mo92othman opened the pull request.

AdebMakhoul reviewed (changes requested):
Good job Mohammed Specially in the Game! :) 

Hope you are doing well, u clearly can work with a good work but we will try to do it great !

In the book List, that not exact what we are looking for, specially in the book card design (red and green design).
Also, the whole page design is vertical, which does not make any sense.
there is a link for a target design I recommend to strict with it, because it contains the basic design rules.
https://hackyourfuture.github.io/example-pages/Browsers/Week1/1-booklist/

AdebMakhoul reviewed (changes requested):
Also, in Cat walk project, any product owner and leader prefer to strict with the requirements, and there is a live target in the homework description,

cat should dance in the middle of the screen for 5 sec and then continue walking, ur cat is dancing while walking :)

mo92othman commented:
Hello, Hope you are doing well,
Thank you for the valuable feedback! I appreciate your guidance. I've fixed the design in the book exercise to meet the exercise requirements.


After studying the API module, I could solve the cat walking exercise in a nicer, cleaner way,(using promises). But I didn't know if I should do that! so I fixed it the hard way since this is an older exercise. But it is working well as required now!

Thank you so much again for your feedback!
